src/edfi-performance-test/README.md

```diff
@@ -29,21 +29,22 @@ tests experience failures.
 
```

Comment:
Should the images above be in the `docs/images` directory instead of `docs/old/images`?